### DISCUSSION TITLE
I have a userscript that opens google trans, but I wanna realize that it does not open with a new TAB every time, but opens there if there is a TAB.

### DISCUSSION THREAD

**Discussion by 1m-N00b** [qutebrowser/qutebrowser#7609] [Q&A]
I have a userscript that opens [google trans](https://github.com/AckslD/Qute-Translate), but I wanna realize that it does not open with a new TAB every time, but opens there if there is a TAB.

**Comment by The-Compiler** [comment]
I have no idea what you're trying to say exactly. The userscript [does](https://github.com/AckslD/Qute-Translate/blob/cd2d201d17bb2d7490700b20d94495327af15e78/translate#L47) open a new tab every time...

**Reply by 1m-N00b** [reply]
Yes. I would like to know how to get the **name?** or **class?** of the tab. 
I also wanna know if what Im planning can do this.
If there is a `**tabs name? or class?**=google trans`, use it again to open it, if not, open it with a new tab.
I wanna realize this kind of branching process
